README.md

```diff
@@ -48,7 +48,7 @@ Do you have many different apps on your Android Phone for the purpose of clickin
 
 * Finally, after all this editing you can easily share the image to your favourite social media sites with our easy-to-use sharing feature.
 
-* Facebook, Twitter, NextCloud, OwnCloud, Imgur, Dropbox, Box, Flickr, Pinterest, Instagram, Whatsapp, and Tumblr - You name it and we have it covered.
+* Messenger, Twitter, NextCloud, OwnCloud, Imgur, Dropbox, Box, Flickr, Pinterest, Instagram, Whatsapp, and Tumblr - You name it and we have it covered.
```

Comment:
Why was Facebook removed?

Reply:
because we got shareTomessenger in our code and sharing on facebook comes in others category of sharing.

Reply:
Please dont remove it